src/config.rs

```diff
@@ -954,6 +954,8 @@ pub struct DbConfig {
     pub raftcf: RaftCfConfig,
     #[online_config(skip)]
     pub titan: TitanDBConfig,
+    #[online_config(skip)]
+    pub sst_ingestion_write_global_seqno: bool,
```

Comment:
Do we necessarily need this config? @yiwu-arbug

Reply:
I was thinking, in case user upgrade from <=2.1 version to >=5.2 version directly, they can set this option to true on upgrade to make sure they can downgrade back to <=2.1 version. But I guess such an upgrade will meet with a ton of other compatibility issues. What do you think?